src/modules/user-settings/components/user-settings.tsx

```diff
@@ -27,7 +27,9 @@ export default function UserSettings() {
 	const { preferredEditor } = useFeatureFlags();
 	const { locale: savedLocale, setLocale: setSavedLocale } = useI18nData();
 
-	const [ isDeletingAllSnapshots, setIsDeletingAllSnapshots ] = useState( false );
+	const activeBulkOperationForUser = useRootSelector( ( state ) =>
+		snapshotSelectors.selectActiveBulkOperationForUser( state, user?.id ?? 0 )
```

Comment:
`?? 0` - it's outside of this PR, but would be cool to refactor and avoid it later. You have bigger picture in your head, will it be difficult to refactor code to either make it optional everywhere, or required, or maybe have conditions e.g. `if ( ! user?.id ) return;` to avoid this kinda "coercion"?

Reply:
It would be unnecessarily difficult to work around this. It might not be crystal clear at first glance, but defaulting to `0` works because there will never be an active operation for a user with that ID.